src/modules/routing/RecipientModule.ts

```diff
@@ -46,6 +48,19 @@ export class RecipientModule {
     this.registerHandlers(dispatcher)
   }
 
+  public async init(connections: ConnectionsModule) {
+    // Check if inviation was provided in config
+    // Assumption: processInvitation is a URL-encoded invitation
+    // TODO Check assumption with config
+    if (this.agentConfig.mediatorInvitation) {
+      const connectionRecord = await connections.receiveInvitationFromUrl(this.agentConfig.mediatorInvitation, {
+        autoAcceptConnection: true,
+        alias: 'InitedMediator', // TODO come up with a better name for this
```

Comment:
```suggestion
        alias: 'InvitedMediator', // TODO come up with a better name for this
```

Reply:
this is a connection created in the initialize method from an invitation provided in the config.
inited short for initialized, and a connection to a mediator, => InitedMediator connection record.
InvitedMediator does capture the sentiment.